catalogue/webapp/pages/api/items.ts

```diff
@@ -0,0 +1,53 @@
+import fetch from 'isomorphic-unfetch';
+import { NextApiRequest, NextApiResponse } from 'next';
+import { ItemsWork, CatalogueApiError } from '@weco/common/model/catalogue';
+import { catalogueApiError } from '../../services/catalogue/common';
+
+function hasOwnProperty<
+  X extends Record<string, unknown>,
+  Y extends PropertyKey
+>(obj: X, prop: Y): obj is X & Record<Y, unknown> {
+  return obj.hasOwnProperty(prop);
+}
+
+export function isCatalogueApiError(
+  response: ItemsWork | CatalogueApiError
+): response is CatalogueApiError {
+  return Boolean(hasOwnProperty(response, 'type') && response.type === 'Error');
+}
+
+async function fetchWorkItems(
+  id: string
+): Promise<ItemsWork | CatalogueApiError> {
+  try {
+    const items = await fetch(
+      `https://api.wellcomecollection.org/catalogue/v2/works/${id}/items`,
```

Comment:
Having just had a look it seems a bit fiddly/out-of-scope but would be good to at some point work out how to get the toggles to here (and the other api endpoint) so we can use `api-stage` etc